docs/tables/aws_config_rule.md

```diff
@@ -0,0 +1,57 @@
+# Table: aws_config_rule
+
+An AWS Config rule represents an AWS Lambda function that you create for a custom rule or a predefined function for an AWS managed rule. The function evaluates configuration items to assess whether your AWS resources comply with your desired configurations. This function can run when AWS Config detects a configuration change to an AWS resource and at a periodic frequency that you choose (for example, every 24 hours).
+
+## Examples
+
+### Basic info
+
+```sql
+select
+    name,
+    rule_id,
+    rule_arn,
+    rule_state,
+    description,
+    created_by,
+    input_parameters,
+    maximum_execution_frequency,
+    scope,
+    source,
+    tags_src,
+    title,
+    tags, 
+    akas,
+    partition,
+    region,
+    account_id
+from
+    aws_config_rule;
+```
+
+### List rules which are in active state
+
+```sql
+select
+    name,
+    rule_id,
+    rule_arn,
+    rule_state 
+from 
+    aws_config_rule;
+where
+    rule_state='ACTIVE'
+```
+
+### List rules which do not have Lambda tag key
```

Comment:
Is this query useful, and the heading is not matching with the actual query, please check once.

Reply:
I have added this query because I found that in one of our table doc.
https://github.com/turbot/steampipe-plugin-gcp/pull/198/commits/8c8c9c5108b2f5dc99fd7ab97fff414298b46b1f#diff-fc16e4e6187f941fc90de368769c621033132f3161a68380ed2ff0c538a080ceR45

But changed the heading from `List rules which do not have Lambda tag key` to `List rules which have Lambda tag key`